Cubical/Algebra/CommRing/RadicalIdeal.agda

```diff
@@ -0,0 +1,90 @@
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.CommRing.RadicalIdeal where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Powerset
+open import Cubical.Foundations.HLevels
+
+open import Cubical.Data.Sigma
+open import Cubical.Data.Sum hiding (map)
+open import Cubical.Data.FinData hiding (elim)
+open import Cubical.Data.Nat renaming ( _+_ to _+ℕ_ ; _·_ to _·ℕ_
+                                      ; +-comm to +ℕ-comm
+                                      ; ·-assoc to ·ℕ-assoc ; ·-comm to ·ℕ-comm
+                                      ; _choose_ to _ℕchoose_ ; snotz to ℕsnotz)
+open import Cubical.Data.Nat.Order
+
+open import Cubical.HITs.PropositionalTruncation
+
+open import Cubical.Algebra.CommRing
+open import Cubical.Algebra.CommRing.Ideal
+open import Cubical.Algebra.CommRing.BinomialThm
+open import Cubical.Algebra.Ring.QuotientRing
+open import Cubical.Algebra.Ring.Properties
+open import Cubical.Algebra.Ring.BigOps
+open import Cubical.Algebra.RingSolver.ReflectionSolving
+
+private
+  variable
+    ℓ : Level
+
+module _ (R' : CommRing ℓ) where
+ private R = fst R'
+ open CommRingStr (snd R')
+ open RingTheory (CommRing→Ring R')
+ open Sum (CommRing→Ring R')
+ open CommRingTheory R'
+ open Exponentiation R'
+ open BinomialThm R'
+ open isCommIdeal
+
+ -- is there a sqrt character?
```

Comment:
``\surd``!

Reply:
ah thanks, I would've never guessed that

Reply:
I happened to know this one, but usually if I don't I look up the unicode online, paste into emacs, and then use `C-u C-x =` to find out how to write it with the Agda input method.